kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/filesystem/InputFormatUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2013 Cloudera Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kitesdk.data.spi.filesystem;
+
+import org.apache.hadoop.mapreduce.RecordReader;
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat;
+import org.kitesdk.compat.DynConstructors;
+import org.kitesdk.data.DatasetDescriptor;
+import org.kitesdk.data.DatasetException;
+
+public class InputFormatUtil {
+  public static final String INPUT_FORMAT_CLASS_PROP = "kite.inputformat.class";
```

Comment:
Yes, we will need to do this in the future as well, but I consider it as separate issue. For that, we may want to have a kite-data-hcatalog module.